src/main/java/org/galatea/starter/service/FinnhubService.java

```diff
@@ -28,4 +35,23 @@ public class FinnhubService {
     public List<FinnhubSymbol> getAllSymbols() {
         return finnhubClient.getAllSymbols();
     }
+
+    public FinnhubQuote getQuote(String symbol) {
+        FinnhubQuoteEntity cached = repository.findById(symbol).orElse(null);
+
+        if (cached != null && cached.getCachedAt().isBefore(cached.getCachedAt().plus(CACHE_TTL))) {
```

Comment:
Will cached.getCachedAt().isBefore(cached.getCachedAt().plus(CACHE_TTL)) ever be false? What is this check for?